hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/ECReplicationConfig.java

```diff
@@ -28,18 +28,29 @@
  * Replication configuration for EC replication.
  */
 public class ECReplicationConfig implements ReplicationConfig {
-  
+
   private static final Pattern STRING_FORMAT = Pattern.compile("(\\d+)-(\\d+)");
```

Comment:
Now we should be able to configure with codec option right? Could you please add comment to show the expected valid format ?